projects/frontend/data-pipelines/gui/projects/ui/src/app/app.module.ts

```diff
@@ -68,7 +69,7 @@ export function lottiePlayerLoader() {
     ],
     declarations: [AppComponent, GettingStartedComponent],
     providers: [
-        { provide: AppConfigService, useClass: AppConfigService, deps: [HttpBackend] },
+        { provide: AppConfigService, useClass: AppConfigService, deps: [HttpBackend, Router] },
```

Comment:
Why are we providing `HttpBackend` instead of `HttpClient`?

Reply:
Wiring the client itself doesn't work for some reason. I tested it locally.